documentation/release-notes/source/2019.2.rst

```diff
@@ -15,6 +15,38 @@ of which are listed below.  For complete details see `the commit logs
 Compiler
 ========
 
+* The default compiler-back-end is now selected by querying the
+  ``SUPPORTED_COMPILER_BACK_ENDS`` build system variable, normally set
+  at configuration time within the ``config.jam`` build script. The
+  default back-end, when it is available, is now LLVM.
+
+* In the LLVM back-end, improved alignment information is supplied to
+  the ``llvm.memcpy`` intrinsic.
+
+* A new ``primitive-the`` primitive function is supported as an
+  annotation that allows the compiler to assume that a value has the
+  given type. These annotations were also added to the ``dylan``
+  library at various frequently-executed expressions where the
+  compiler is otherwise unable to infer the type, reducing the number
+  of run-time type checks when using the LLVM back-end.
+
+* Optimization for cases where the ``instance?`` function is used in
+  an ``if`` or ``case`` condition has been improved.
+
+* A bug in the LLVM back-end that could cause stack overflows in
+  long-running functions has been fixed.
+
+* The C back-end now uses C99 hexadecimal floating-point constants in
+  its output to prevent loss of precision due to decimal conversion.
+
+* A `longstanding bug
+  <https://github.com/dylan-lang/opendylan/issues/1068>` in an
```

Comment:
All of your links need a trailing underscore.  (Easy to forget...one little reason I like Markdown better.)